homeassistant/components/mastodon/strings.json

```diff
@@ -11,10 +11,20 @@
         "data_description": {
           "base_url": "The URL of your Mastodon instance."
         }
+      },
+      "reauth_confirm": {
+        "description": "Please reauthenticate with Mastodon.",
+        "data": {
+          "client_id": "Client Key",
+          "client_secret": "Client Secret",
```

Comment:
You can use references to the previous one 
`[%key:component::mastodon::config::step::user::data::client_id%]`
I probably forgot it last time, but can you decapitalize `Key` and `Secret`?